spec/chewy/multi_search_spec.rb

```diff
@@ -0,0 +1,85 @@
+require 'spec_helper'
+require 'chewy/multi_search'
+
+describe Chewy::MultiSearch do
+  before { Chewy.massacre }
+
+  before do
+    stub_model(:city)
+    stub_model(:country)
+
+    stub_index(:places) do
+      define_type City do
+        field :name, **KEYWORD_FIELD
+        field :country_id, **KEYWORD_FIELD
+
+        def self.aggregate_by_country
+          aggs(country: {terms: {field: :country_id}})
+        end
+      end
+    end
+  end
+
+  let(:places_query) { PlacesIndex.all }
+
+  describe '#queries' do
+    specify 'returns the queries that are a part of the multi search' do
+      multi_search = described_class.new([places_query])
+      expect(multi_search.queries).to contain_exactly(places_query)
+    end
+  end
+
+  describe '#add_query' do
+    specify 'adds a query to the multi search' do
+      multi_search = described_class.new([])
+      expect do
+        multi_search.add_query(places_query)
+      end.to change {
+        multi_search.queries
+      }.from([]).to([places_query])
+    end
+  end
+
+  context 'when given two queries' do
+    let(:queries) { [aggregates, results] }
+    let(:aggregates) { PlacesIndex::City.aggregate_by_country.limit(0) }
```

Comment:
Let's remove the type City, it will work and we won't have to change it for ES6

Reply:
Ok! Done!